extension/apmconfigextension/opamp_callbacks_test.go

```diff
@@ -177,17 +177,6 @@ func TestOnMessage(t *testing.T) {
 					expectedServerToAgent: &protobufs.ServerToAgent{
 						InstanceUid:  []byte("test"),
 						Capabilities: uint64(protobufs.ServerCapabilities_ServerCapabilities_OffersRemoteConfig),
-						RemoteConfig: &protobufs.AgentRemoteConfig{
```

Comment:
Should we also add a test case with the failure case you shared?

Reply:
I could do so, yes. Though this "APPLYING" test case happens to test a similar case. Adding one for `RemoteConfigStatuses_FAILED` is a good idea.

Reply:
commit c277b97